packages/types/src/types/authentication-service.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './user';
+import { DatabaseInterface } from './database-interface';
+
+// import AccountsServer from '@accounts/server';
```

Comment:
Not possible to do that? would be better for type checking

Reply:
I didn't succeed... it makes a circular dependency and the suite can't compile

Reply:
Okay let's keep it like that for now we can still fix it later

Reply:
Can you just add a TODO as a reminder and explain the pb as comments?